main.go

```diff
@@ -252,6 +254,9 @@ func main() {
 	kingpin.HelpFlag.Short('h')
 	kingpin.Parse()
 
+	log.Base().SetLevel(*logLevel)
```

Comment:
The whole point of flags is that they appear without effort if a upstream library adds them, just like metrics. Having to duplicate code like this in 10s to 100s of repositories is not viable.